site/docs/plugins/commands.md

```diff
@@ -5,10 +5,628 @@ next: false
 
 # Commands (`commands`)
 
-Coming soon, please come back later.
+Command handling on steroids.
+
+This plugin provides various features related to command handling that are not contained in the [command handling done by the core library](../guide/commands).
+Here is a quick overview of what you get with this plugin:
+
+- Better code readability by encapsulating middleware with command definitions
+- User command menu synchronization via `setMyCommands`
+- Improved command grouping and organization
+- Ability to scope command reach, e.g: only accessible to group admins or
+  channels, etc
+- Defining command translations
+- `Did you mean ...?` feature that finds the nearest existing command to a given
+  user miss-input
+- Case-insensitive command matching
+- Setting custom behavior for commands that explicitly mention your bot's user,
+  like: `/start@your_bot`
+- Custom command prefixes, e.g: `+`, `?` or any symbol instead of `/`
+- Support for commands that are not in the beginning of the message
+- RegExp Commands!
```

Comment:
We do not split sentences like that.
We always put the entire sentence per line.
The IDE may perform any wraps, but it should not affect the actual sentence layout.

```suggestion
- Better code readability by encapsulating middleware with command definitions
- User command menu synchronization via `setMyCommands`
- Improved command grouping and organization
- Ability to scope command reach, e.g: only accessible to group admins or channels, etc
- Defining command translations
- `Did you mean ...?` feature that finds the nearest existing command to a given user miss-input
- Case-insensitive command matching
- Setting custom behavior for commands that explicitly mention your bot's user, like: `/start@your_bot`
- Custom command prefixes, e.g: `+`, `?` or any symbol instead of `/`
- Support for commands that are not in the beginning of the message
- RegExp Commands!
```